packages/clippy/src/ClippyProvider.jsx

```diff
@@ -48,6 +46,7 @@ const ClippyProvider = ({ children, agentName = AGENTS.CLIPPY }) => {
         () => {
           setClippy();
         },
+        window.CLIPPY_CDN || 'https://gitcdn.xyz/repo/pi0/clippyjs/master/assets/agents/',
```

Comment:
I'd also let the condition there because on SSR frameworks, like NextJS, it would throw an error saying `window` doesn't exists
```suggestion
        typeof window !== "undefined" && window.CLIPPY_CDN ? window.CLIPPY_CDN : 'https://gitcdn.xyz/repo/pi0/clippyjs/master/assets/agents/',
```

WDYT?

Reply:
Good point. I think since we are not trying to assign a value to `window.CLIPPY_CDN` anymore, we could probably also just get away with this solution:

```suggestion
        window?.CLIPPY_CDN || 'https://cdn.jsdelivr.net/pi0/clippyjs/master/assets/agents/',
```
Btw, I'm totally fine with going for jsdelivr. Especially since this will allow the URL to be overwritten by users.